ddtrace/contrib/anthropic/__init__.py

```diff
@@ -0,0 +1,98 @@
+"""
+The Anthropic integration instruments the Anthropic Python library to traces for requests made to the models for messages.
+
+All traces submitted from the Anthropic integration are tagged by:
+
+- ``service``, ``env``, ``version``: see the `Unified Service Tagging docs <https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging>`_.
+- ``anthropic.request.model``: Anthropic model used in the request.
+- ``anthropic.request.api_key``: Anthropic API key used to make the request (obfuscated to match the Anthropic UI representation ``sk-...XXXX`` where ``XXXX`` is the last 4 digits of the key).
+- ``anthropic.request.parameters``: Parameters used in anthropic package call.
+
+
+(beta) Prompt and Completion Sampling
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following data is collected in span tags with a default sampling rate of ``1.0``:
+
+- Prompt inputs and completions for the ``Messages.create`` endpoint.
+
+
+Enabling
+~~~~~~~~
+
+The Anthropic integration is enabled automatically when you use
+:ref:`ddtrace-run<ddtracerun>` or :ref:`import ddtrace.auto<ddtraceauto>`.
+
+Note that these commands also enable the ``requests`` and ``aiohttp``
+integrations which trace HTTP requests from the Anthropic library.
```

Comment:
Question, which request libraries are used by Anthropic? Just want to make sure this is an accurate statement

Reply:
httpx, should I remove this?

Reply:
httpx does both async and sync